feature-libs/asm/components/public_api.ts

```diff
@@ -1 +1,9 @@
-export * from './index';
+export * from './asm-components.module';
+export * from './asm-main-ui/asm-main-ui.component';
+export * from './asm-session-timer/asm-session-timer.component';
+export * from './asm-session-timer/format-timer.pipe';
+export * from './asm-toggle-ui/asm-toggle-ui.component';
+export * from './csagent-login-form/csagent-login-form.component';
+export * from './customer-emulation/customer-emulation.component';
+export * from './customer-selection/customer-selection.component';
+export * from './services/index';
```

Comment:
Just curious: why remove the `index.ts` and move all the exports in `public_api.ts` ?

Reply:
We have `public_api.ts`, which `export * from index`. The ng-packagr v14 throws error "Entry point 'xxxx' has an 'index.ts' parallel to the 'entryFilePath'. The 'entryFilePath' should be updated to point to the 'index.ts' file." .  (https://github.com/ng-packagr/ng-packagr/pull/2349)

There are 2 ways to fix it: 
1. remove `public_api.ts`, and make `entryFile` in `ng-package.json` points to `index.ts`; 
2. move the content of `index.ts` to `public_api.ts`, and remove `index.ts`. 

I prefer 2, because we use `public_api` in all entry points; and I also tried 1, it does not work (The error is "index.ts is missing from typescript compilation"), ng-packagr v14 automatically generates index.d.ts.